apps/coscrad-frontend/src/components/resources/spatial-features/polygon-spatial-feature-detail.full-view.presenter.tsx

```diff
@@ -0,0 +1,48 @@
+import { ISpatialFeatureViewModel, ResourceType } from '@coscrad/api-interfaces';
+import {
+    FloatSpacerDiv,
+    SinglePropertyPresenter,
+} from 'apps/coscrad-frontend/src/utils/generic-components';
+import { CoscradLeafletMap } from './leaflet';
+import styles from './polygon-spatial-feature-detail.full-view.presenter.module.scss';
+import { SpatialFeatureDetailThumbnailPresenter } from './thumbnail-presenters';
+import { MultiPolygon2D, Position2D } from './types';
+
+export const PolygonSpatialFeatureDetailFullViewPresenter = (
+    spatialFeature: ISpatialFeatureViewModel
+): JSX.Element => {
+    const { id, geometry, properties } = spatialFeature;
+    const { type, coordinates } = geometry;
+    const { name, description, imageUrl } = properties;
+
+    /**
+     * TODO: find centre of polygon rather than first point of first linear ring
+     */
+    const firstLinearRing = [...(coordinates as MultiPolygon2D)].shift();
+    const initialCentreCoordinates = firstLinearRing[0].map((line2D) => line2D) as Position2D;
+
+    return (
+        <>
+            <div className={styles['map']}>
+                <CoscradLeafletMap
```

Comment:
Inject this so that we don't have to rewrite the logic if we want to use a different map. Program to `ICoscradMap`.